lib/match_table.rb

```diff
@@ -42,27 +42,30 @@ module MatchTable
         @actual_rows.each do |actual_row|
           matched_rows << match_row(header_positions:, actual_row:)
         end
+      rescue Capybara::ExpectationNotMet => e
```

Comment:
I'm intentionally transforming the exception into something external to Capybara.